src/reflect/scala/reflect/io/ZipArchive.scala

```diff
@@ -34,7 +35,7 @@ import ZipArchive._
  *  ''Note:  This library is considered experimental and should not be used unless you know what you are doing.''
  */
 object ZipArchive {
-  private[io] val closeZipFile = sys.props.get("scala.classpath.closeZip").map(_.toBoolean).getOrElse(false)
+  private[io] val closeZipFile = booleanProperty("scala.classpath.closeZip")
```

Comment:
Are the semantics of Boolean.getBoolean and String.toBoolean the same?

Reply:
ok close enough

![image](https://user-images.githubusercontent.com/344610/149921525-76ced3ac-dc5e-4dd0-b17f-b0b1b3a73855.png)

Reply:
less throwy...